### PR TITLE
Add step and form test while fixing test dependencies

### PR DESCRIPTION
**Pull request by casz**

### CODE REVIEW

**Comment by casz** [comment]
This will most likely fail, I properly need to rebase onto #24

**Comment by casz** [comment]
Rebaed, so merge order is #24 and then #31

File: src/main/resources/jenkins/plugins/office365connector/Webhook/config.jelly [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            </f:entry>
+
+            <f:entry title="Notify Aborted" field="notifyAborted">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Failure" field="notifyFailure">
+                <f:checkbox />
+            </f:entry>
+
```
Comment (SrhebbarMicrosoft) [inline src/main/resources/jenkins/plugins/office365connector/Webhook/config.jelly]: I think these needs to be changed... You updated in your previous PR... This is reverting those changes.

Comment (casz) [reply src/main/resources/jenkins/plugins/office365connector/Webhook/config.jelly]: I need to rebase onto master, that's all

**Comment by casz** [comment]
@SrhebbarMicrosoft ready for review after rebase 👍

**Pull request merged**